ui-tests/cypress/e2e/components/ux_fragments.cy.js

```diff
@@ -527,14 +527,16 @@ describe('UX Fragments', () => {
                 }));
       });
 
-      it([Tag.SANITY, 'ENG-3522'], 'Deleting a fragment', () => {
+      it([Tag.SANITY, 'ENG-3522', 'ENG-3952'], 'Deleting a fragment', () => {
         cy.get('@fragmentsToBeDeleted').then(fragments => fragments[0]).then(fragment =>
             openUXFragmentsPage()
                 .then(page => page.getContent().getKebabMenu(fragment.code).open().clickDelete())
                 .then(page => page.getDialog().confirm())
                 .then(page => {
                   cy.validateToast(page, fragment.code);
                   page.getContent().getTableRows().should('not.contain', fragment.code);
+                  page.getContent().getPagination().getItemsCurrent().invoke('text').should('be.equal', '1-10');
```

Comment:
Use `have.text` in the should, instead of _invoking_ text and then assert it to _be equal_